client/document/index.jsx

```diff
@@ -79,6 +80,9 @@ class Document extends React.Component {
 			'jetpack-connect' === sectionName &&
 			'woocommerce-onboarding' === requestFrom;
 
+		const isJetpackCloudFlow =
+			config.isEnabled( 'jetpack-cloud' ) && 'jetpack-cloud' === sectionName;
```

Comment:
Is the `config.isEnabled` check really useful here? I'd say the section name check is sufficient.

If you agree, maybe we could remove the `const` altogether and move the check inline as it's simple enough.

Reply:
It makes sense to me. I've based this check on the [existing check for `isJetpackWooCommerceFlow` a few lines above](https://github.com/Automattic/wp-calypso/pull/39172/files#diff-3fb371c23faa311830bf33b7e7e855a3R78), but - really - we can rely on the section name since it's determined by the same config flag anyway.

Reply:
Not only we can rely on the section name, but it also will make sense if we ever decide to enable multiple sections for the Jetpack Cloud and we run it alongside Calypso sections.